### DISCUSSION TITLE
O que você está aprendendo agora?

### DISCUSSION THREAD

**Discussion by TailoMateus** [frontendbr/forum#1199] [Discussão]
Olá, fiz essa mesma pergunta no dev.to 
https://dev.to/tailomateus/what-are-you-learning-right-now-15bd

E aí, o que você está aprendendo neste momento?

**Comment by kivervinicius** [comment]
React and React Native

**Comment by Gpaula95** [comment]
Node API Rest e mongo db

**Comment by ViniciusGularte** [comment]
React, Redux e React Native, mas principalmente tomei vergonha na cara kk e to estudando Javascript a fundo a partir desse livro [You-Dont-Know-JS](https://github.com/cezaraugusto/You-Dont-Know-JS), se alguém souber outros livros bons de Javascript  fico grato.

**Comment by Roger-Melo** [comment]
- React
- Git 

com o melhor professor, Fernando Daciuk. 
Indico fortemente os cursos 

- [JS Ninja](https://www.udemy.com/curso-javascript-ninja/)
- [React Ninja](https://www.udemy.com/curso-reactjs-ninja/)
- [Git e GitHub Ninja](https://www.udemy.com/git-e-github-ninja/)

**Comment by robbporto** [comment]
Ando aprendendo Ruby on Rails!

**Comment by ninetails** [comment]
Frogramação Puncional '-'~

**Comment by lussolli** [comment]
C#, ASP.NET Core, Vue.Js e Git.

**Comment by kvnol** [comment]
Vue.js

**Comment by tunnes** [comment]
React e revendo o básico do ruby, @ViniciusGularte recentemente também passei a me questionar se realmente conheço JavaScript xD

**Comment by mateusmiguel** [comment]
matemática

**Comment by igorthurow** [comment]
Principalmente React com TypeScript. Estudando conceitos de orientação a objeto que estão fazendo falta em um projeto. Inglês sempre :)

**Comment by thismarcoantonio** [comment]
Programação funcional

**Comment by valterbarros** [comment]
@mateusmiguel fiquei curioso porque matemática, algum motivo em especial?

**Comment by kavalcante** [comment]
Design de componentes avançados em Vue.js (enquanto crio uma lib de componentes)

**Comment by vitocaio** [comment]
React, Redux

**Comment by lucasferreiralimax** [comment]
Criando CLI com node.js e estudando inglês.

**Comment by wesmelo** [comment]
node com express e react.js

**Comment by caioincau** [comment]
Gestão e inglês, como assumi a liderança de uma equipe e ela se tornou global, estou focando nessas skills.

**Comment by Adriano-Web** [comment]
JavaScript. Na verdade já tinha aprendido o básico dele, mas agora quero dominar o puro por completo e depois aprender suas bibliotecas e frameworks.

**Comment by lucasferreiralimax** [comment]
React + Webpack Curso do @fdaciuk ;-)

**Comment by gabrieldarezzo** [comment]
VueJS, aprofundando o Laravel, nas horas vagas brincando com Rasberry / Eletrônica

**Comment by gabrieldarezzo** [comment]
@Adriano-Web tenho uma lista de [exercícios simples](https://github.com/gabrieldarezzo/helpjs-ravi) em JS / JS-Vanilla.
Todos focados no JS puro, recentemente adicionei alguns exercícios de ES6 & princípios de animação

**Comment by praganavictor** [comment]
TypeScript, Angular e Deploy

**Comment by didiraja** [comment]
React e WP ao mesmo tempo, mas dese já com olho em Node

**Comment by waldemirflj** [comment]
Vue.js, GraphQL e Gestão/Liderança.

**Comment by Adriano-Web** [comment]
@gabrieldarezzo Obrigado, vou olhar sim.

**Comment by tdcosta100** [comment]
C# eu já sei há tempos. Mas programação em tempo real com multithread é OUTRO nível de dificuldade. Tem sido um aprendizado enorme.

**Comment by mosesmeirelles** [comment]
React e Django (Python)

**Comment by mdandrade** [comment]
React, flertes em UX/UX e planejando Django.

**Comment by mutheusalmeida** [comment]
JavaScript Basics, Svelte, UX / UI Design

**Comment by adamfoerster** [comment]
[Svelte](https://discord.gg/Xf9zzW9)
Bash
Firebase

**Comment by jpferreiradev** [comment]
HTML 
CSS 
Java

**Comment by ninetails** [comment]
(atualizando quase 1 ano depois :rofl:)

Svelte(++) 
RxJS

**Comment by andrelsa** [comment]
Não sou front-end, trabalho como back-end mas estou querendo aumentar minhas stacks, estudando JS puro, posteriormente pretendo focar em algum framework. Não sei se é a melhor abordagem pra aprimorar o meu nível de front-end, mas prefiro trilhar esse caminho.

No momento estou estudando: Além do back-end (Java, Spring Boot e Banco de Dados), no front-end (HTML, CSS e JS).

**Comment by felipefialho** [comment]
@ninetails Consegue abrir uma issue sobre o Svelte e contar como está sendo sua experiência? Muito me interessa.

**Comment by ninetails** [comment]
@felipefialho mals ter escolhido um outro formato:

https://dev.to/ninetails/experiencias-com-svelte-por-um-desenvolvedor-react-pt-br-4mj9

Ficou meio longo pq como eu nunca posto artigos... mesmo assim, falando sobre minha experiência foquei em ferramental, já que vi que ainda não é algo tão explorado nos artigos sobre o assunto (geralmente são introdutórios ou tutoriais)

Se quiser, posso abrir uma issue resumindo. E desculpe a demora em fazê-lo tb.

**Comment by will3g** [comment]
NodeJS, React, React Native e Docker.

**Comment by diessica** [comment]
música

**Comment by tdcosta100** [comment]
Algoritmos genéticos (para alinhamento de sequências, porém nada a ver com genética hehehe). Inclusive, isso rendeu um código bacana em C#:

https://gist.github.com/tdcosta100/3cd0c3298c072815568edbf9d74166b5

**Comment by Emerson-developer** [comment]
Atualmente estou escolhendo entre C/C++ ou as tecnologias de Javascript (React, ReactNative, NodeJS)...queria focar em desenvolvimento mobile.

**Comment by felipe-heredia** [comment]
Atualmente estou bem indeciso no que pegar firme, porém, minhas principais metas são Nodejs, ReactJS e ReactNative. Amei muito esses tecnologias apenas de serem minhas primeiras.
